### PR TITLE
feat: add translation for german

### PR DESCRIPTION
**Pull request by theexiile1305**
This adds the translation for German.

### CODE REVIEW

File: src/assets/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "locale": "de"
+    },
+    "cta-button-placeholder": "Zugriff über Ihre Institution",
+    "cta-button-header": "Zugriff über",
+    "cta-link": "Institution hinzufügen oder ändern",
+    "ds-header": "Zugriff auf",
+    "ds-footer-about-us": "Über uns",
+    "ds-search-heading": "Finden Sie Ihre Institution",
+    "ds-search-subheading": "Ihre Universität, Organisation oder Firma",
+    "ds-search-example": "Beispiele: Science Institute, Lee@uni.edu, UCLA",
```
Comment (leifj) [inline src/assets/translations/de.json:16]: would it make sense to have examples that resonate with German-speaking users instead?

Comment (theexiile1305) [reply src/assets/translations/de.json]: Yea, that makes sense. I've updated the PR.

**Comment by theexiile1305** [comment]
@leifj @skoranda @mrvanes @fredrikdomeij This PR is ready for review with the code review remarks — many thanks for considering my request.

**Review by leifj — APPROVED** [review]
Looks good to me... with my limited knowledge of good german

**Pull request merged**